### ISSUE TITLE
Disabling auto-complete in the "Create New Directory" dialog?

### ISSUE THREAD

**Issue by fdsjhbdaskhdas** [doublecmd/doublecmd#544]
Hi!

In the "Create New Directory" (F7) dialog there is some auto-complete.
If you created a directory, lets say, "fdsjhbdaskhdas", some time ago,
and now you want to make a directory "fd", so you enter "fd", and press the enter-key,
you unintendently get the directory "fdsjhbdaskhdas" created.

I understand that it can be convenient to create several dirs with similar names, but most times it is (at least for me) nasty and sometimes even dangerous.

I searched, but could not find how to (optionally) turn off this behavior. Can it be done?

If not, please could you implement an option to disable this auto-completion? Ideally by a checkbox directly in the auto-complete dialog?
If disabled, "fdsjhbdaskhdas" could been offered  after typing "fd", but on pressing Enter, the directory "fd" gets created, unless for example the Tab key or cursor-down or the like is pressed prior to Enter.

Thanks in advance!

Double Commander
Version: 1.0.6 beta
Revision: 120
Commit: b1d0f2f15
Build date: 2022/06/07
Lazarus: 2.2.0.4
FPC: 3.2.2
Platform: x86_64-Linux-gtk2
OS version: Debian bookworm/sid

Widgetset library: GTK 2.24.33

**Comment by ags1234** [comment]
I agree, I happened to create directories with unwanted names because of this feature.
Maybe it could be implemented as an option?

**Comment by LifeCanvasDev** [comment]
I agree - there should be an option to disable this! It really is annoying and sometimes outright frustrating!

Ironically, Double Commander does *not* have the useful feature that Total Cmdr has of offering autocomplete when typing a directory name to move a file or directory to... e.g. If I press F6 in Total Cmdr and type ..\na it will show me available directory names in the parent directory, such as ..\name-of-parent-dir
